package.json

```diff
@@ -4,9 +4,10 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "test": "echo \"Currently No Tests\"",
+    "test": "mocha dist/test/unit/**/*.js",
```

Comment:
If the unit tests are being built into dist then, when the application is deployed the unit tests will end up being deployed with the production code, which is a behaviour we would want to avoid